backend/apps/DatabaseForm/serializers.py

```diff
@@ -0,0 +1,15 @@
+from rest_framework import serializers
+
+from backend.apps.DatabaseForm.models import TelegramUser, TelegramTask
+
+
+class TelegramUserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = TelegramUser
+        fields = ['telegram_id']
```

Comment:
Лучше использовать кортеж
```suggestion
        fields = ('telegram_id',)
```